src/OktaEntity.php

```diff
@@ -14,9 +14,13 @@ public function __construct(private array $acceptedAttributes)
 
     public function fill(array $attributes)
     {
-        foreach ($attributes as $attribute) {
-            $this->{strtolower($attribute->getName())} =  $attribute->getFirstAttributeValue();
-        }
+	    foreach ($attributes as $attribute) {
+		    $this->{strtolower($attribute->getName())} = $attribute->getFirstAttributeValue();
+
+		    if (str_contains($attribute->getNameFormat(), 'format:unspecified')) {
```

Comment:
@rexs123 lets move the conditional to happen first with an early return here please

Reply:
Abstracted to a private method.